embassy-rp/src/gpio.rs

```diff
@@ -953,7 +965,7 @@ macro_rules! impl_pin {
         impl SealedPin for peripherals::$name {
             #[inline]
             fn pin_bank(&self) -> u8 {
-                ($bank as u8) * 32 + $pin_num
+                ($bank as u8) * 64 + $pin_num
```

Comment:
if bank is bit 7, shouldn't this be 128?

Reply:
Ah, of course it is. I just didn't hit it because I'm not trying to use the qspi pins. The way pins are selected is a bit fiddly imo. When I get around to trying the gpio co-processor it should improve.